File: shopBackend/src/main/java/com/example/shop/dtos/request/ProductRequest.java
package com.example.shop.dtos.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProductRequest {
    private String description;
    private Integer price;
    private Integer amount;
    private String picture;
    private String nameOfProduct;

}

File: Android app/app/src/main/java/com/example/shop/data/dtos/request/ProductRequest.kt
package com.example.shop.data.dtos.request

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

data class ProductRequest (
    val description: String,
    val price: Int,
    val amount: Int,
    val picture: String,
    val nameOfProduct: String
)